docs/UserGuide.md

```diff
@@ -9,32 +9,57 @@ title: User Guide (v1.3)
 
 --------------------------------------------------------------------------------------------------------------------
 
-## Introduction
+## Introduction - Overview of the application
 
 #### What is TrackAScholar?
 
-TrackAScholar (TAS) is the **only** desktop app any university administrative staff needs for managing scholarship applications.
-TAS helps administrative staff manage scholarship applications by storing the applicant's contact details, scholarship applied and majors taken all in one place.
+TrackAScholar (TAS) helps university administrative staff manage scholarship applications.
+It stores all applicant's details such as contact info and scholarship applied etc. all in one place.
+TrackAScholar (TAS) is a one of a kind desktop app.It is the **only** application that any university administrative staff needs.
 
-TAS  presenting the administrative staff with the option to pin important applicants, edit their details easily,
- find or filter for applicants with ease and otherwise speeds up the application process.
+TAS have many functions that university administrative staff will find to be useful.
+
+These functions include:
+1. Pinning of important applicants. 
+2. Easy edit of applicant's detail.
+For example, you typed in the wrong name.You can easily edit the wrong name to the correct one.
+3. Finding a particular applicant. 
```

Comment:
Finding a particular applicant **by a certain trait**

Reply:
This has been resolved.

Thank you.